addons/sale/models/sale.py

```diff
@@ -396,6 +396,7 @@ def action_invoice_create(self, grouped=False, final=False):
         :param final: if True, refunds will be generated if necessary
         :returns: list of created invoices
         """
+        import pudb; pu.db
```

Comment:
@tbe-odoo don't forget to remove this before merging it in.

Reply:
Woops committed the wrong thing, thanks :)

Reply:
You're welcome, suspected it was a mistake. 😉